src/components/CommandBar.js

```diff
@@ -102,37 +106,49 @@ const CommandBar = React.createClass({
 
   componentDidMount() {
     const shortcuts = this.context.shortcuts;
-    const handleEvent = (e, func) => {
+    const handlePressAnimation = button => {
+      if (button) {
+        button.style.opacity = "0";
+        button.style.transform = "scale(1.3)";
+        setTimeout(() => {
+          button.style.opacity = "1";
+          button.style.transform = "none";
+        }, 200);
+      }
+    };
+
+    const handleEvent = (e, func, button) => {
       e.preventDefault();
       e.stopPropagation();
+      handlePressAnimation(button);
       func();
     };
 
     COMMANDS.forEach((action) => shortcuts.on(
       getKey(action),
-      (_, e) => handleEvent(e, this.props[action]))
+      (_, e) => handleEvent(e, this.props[action], this[action]))
     );
 
     if (isMacOS) {
       // The Mac supports both the Windows Function keys
       // as well as the Mac non-Function keys
       COMMANDS.forEach((action) => shortcuts.on(
         getKeyForOS("WINNT", action),
-        (_, e) => handleEvent(e, this.props[action]))
+        (_, e) => handleEvent(e, this.props[action], this[action]))
       );
     }
   },
 
   renderStepButtons() {
     const className = this.props.pause ? "active" : "disabled";
     return [
-      debugBtn(this.props.stepOver, "stepOver", className,
+      this.debugBtn(this.props.stepOver, "stepOver", className,
         L10N.getFormatStr("stepOverTooltip", formatKey("stepOver"))
       ),
-      debugBtn(this.props.stepIn, "stepIn", className,
+      this.debugBtn(this.props.stepIn, "stepIn", className,
         L10N.getFormatStr("stepInTooltip", formatKey("stepIn"))
       ),
-      debugBtn(this.props.stepOut, "stepOut", className,
+      this.debugBtn(this.props.stepOut, "stepOut", className,
```

Comment:
something like this on the instance?
  saveButtonRef(type) {
    this[type] = button;
  }

@jasonLaster